src/devices/bus/ss50/mps.cpp

```diff
@@ -48,26 +50,34 @@ class ss50_mps_device : public device_t, public ss50_card_interface
 
 private:
 	DECLARE_WRITE_LINE_MEMBER(acia_irq_w);
+	DECLARE_WRITE_LINE_MEMBER(route_cts_r);
 
 	required_device<acia6850_device> m_acia;
 	required_ioport m_irq_jumper;
 	required_ioport m_rate_jumper;
+	required_ioport m_cts_route;
 };
 
 
 static INPUT_PORTS_START( mps )
 	PORT_START("IRQ")
-	PORT_DIPNAME(1, 0, "IRQ")
-	PORT_DIPSETTING(1, DEF_STR(Off))
-	PORT_DIPSETTING(0, DEF_STR(On))
+	PORT_CONFNAME(1, 0, "IRQ")
+	PORT_CONFSETTING(1, DEF_STR(Off))
+	PORT_CONFSETTING(0, DEF_STR(On))
 
 	PORT_START("BAUD")
-	PORT_DIPNAME(0x1f, 0x1d, "Baud Rate")
-	PORT_DIPSETTING(0x1e, "110")
-	PORT_DIPSETTING(0x1d, "150 / 9600")
-	PORT_DIPSETTING(0x1b, "300")
-	PORT_DIPSETTING(0x17, "600")
-	PORT_DIPSETTING(0x0f, "1200")
+	PORT_CONFNAME(0x1f, 0x1d, "Baud Rate")
+	PORT_CONFSETTING(0x1e, "110")
+	PORT_CONFSETTING(0x1d, "150 / 9600")
+	PORT_CONFSETTING(0x1b, "300")
+	PORT_CONFSETTING(0x17, "600")
+	PORT_CONFSETTING(0x0f, "1200")
+
+	PORT_START("CTS_ROUTE")
+	PORT_CONFNAME(1, 0, "CTS route")
+	PORT_CONFSETTING(0, "Wired low (standard)")
+	PORT_CONFSETTING(1, "Connected through (modified)")
```

Comment:
This is another control line that can get stuck if the switch is flipped while running.  I won't hold up merging it any longer, but this one should be fixed. as well.